src/slacko.ml

```diff
@@ -599,12 +599,16 @@ let optionally_add key value uri = match value with
 
 let definitely_add key value = optionally_add key (Some value)
 
-let unauthed_endpoint ?(base_url=default_base_url) e =
+let unauthed_endpoint ~base_url e =
```

Comment:
Why was this change necessary? It looks to me like it should've worked out of the box.